### DISCUSSION TITLE
Enquete: Opinião na moderação do fórum

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2291] [Polls]
Olá tudo bem? Esta é a primeira enquete que tem aqui no frontendbr/forum e como maintainer preciso pedir a opinião de vocês!

### Qual o problema?
Algumas discussões não tem fim eminente e duram muitos anos e continuarão existindo para por um bom tempo:

4 exemplos de discussões que se enquadram:
- https://github.com/frontendbr/forum/discussions/28
- https://github.com/frontendbr/forum/discussions/56
- https://github.com/frontendbr/forum/discussions/124
- https://github.com/frontendbr/forum/discussions/602

### Solução paliativa:
Quando atingir 100 comentários ou mais, e tempo de inatividade.
Vamos fazer lock para não permitir novos comentários na mesma thread.

### Objetivo:
Informar pessoas interessadas no tema para criar novas discussões dentro do assunto fazendo links entre elas e vir com novas informações relevantes para ajudar no desenvolvimento de cada caso.

### Conclusão da votação:
- Apoio: 1 ponto
- Não aponto -1 ponto
- Apoio com ressalvas: 0.5 ponto
- Abster do voto: 0

Vamos somar o total de votos de acordo com a pontuação e se o resultado for maior que 0 `resultado > 0` irei passar atuar no bloqueio de novos comentários nas threads mencionadas acima.

----

Isso é apenas uma tentativa de trazer a atenção de vocês que o fórum está vivo e que fazemos eles juntos, e que se você está insatisfeito com algo, crie um tópico para discutir e vamos aprimora-lo sempre.

Obrigado.

**Comment by bpires** [comment]
Cheguei à pouco, mas me parece que o fórum anda meio parado. A ideia é muito boa, mas será que a plataforma talvez não seja a mais adequada para manter o engajamento e integração entre o pessoal? De qualquer forma, parabéns pela iniciativa de vocês!

**Reply by felquis** [reply]
Caso queira participar mais, faça inscrição para receber notificações de todas as novas postagens, dessa forma você consegue ver se pode ajudar em novas perguntas também.
